src/package.rs

```diff
@@ -333,6 +333,12 @@ impl Eq for Package {}
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Packages(pub Vec<Package>);
 
+impl From<Vec<Package>> for Packages {
+    fn from(x: Vec<Package>) -> Self {
+        Packages { 0: x }
```

Comment:
The `Packages(x)` syntax is better here.

Reply:
Fixed

Reply:
Done